stripe/src/main/java/com/stripe/android/model/PaymentMethodOptionsParams.kt

```diff
@@ -9,7 +9,7 @@ sealed class PaymentMethodOptionsParams(
 
     @Parcelize
     data class Card(
-        val cvc: String? = null
+        var cvc: String? = null
```

Comment:
why this change?

Reply:
All properties on param classes should be `var` to make it easier for users to work with these classes. Properties that represent Stripe objects (i.e. objects that are returned by the Stripe API) should continue to be `val`. This matches `stripe-ios`'s behavior.